recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python >=3.6
+    - python
     - python-dateutil >=2.7.0
     - typing_extensions
```

Comment:
You could restrict typing_extensions to py < 3.8
https://github.com/arrow-py/arrow/blob/1.2.3/setup.py#L27
`typing_extensions #[py<38]`